scripts/upgrade

```diff
@@ -146,14 +146,19 @@ ynh_install_php --phpversion="$YNH_PHP_VERSION" --package="$extra_pkg_dependenci
 #=================================================
 # NGINX CONFIGURATION
 #=================================================
-ynh_script_progression --message="Upgrading nginx web server configuration..." --weight=2
+ynh_script_progression --message="Upgrading nginx web server configuration..." --weight=3
 
 ynh_backup_if_checksum_is_different --file="/etc/nginx/conf.d/$domain.d/$app.conf"
 
 # Delete current nginx configuration to be able to check if .well-known is already served.
-ynh_remove_nginx_config $YNH_PHP_VERSION
+ynh_remove_nginx_config
 ynh_app_setting_delete --app=$app --key="checksum__etc_nginx_conf.d_$domain.d_$app.conf"
 
+ynh_systemd_action --service_name=nginx --action=reload
```

Comment:
There's already a reload into the helper.
If you really want to add a reload here, use the log detection in replacement of the sleep.

Reply:
Ho sorry I forgot about that. Maybe just a sleep then to be sure? I can't reproduce 100% the issue --'

Reply:
I guess 1s isn't that much, but using a sleep is hardly a reliable fix

Reply:
You're right, I did this quick and dirty fix because we have the same here:
https://github.com/YunoHost/yunohost/blob/00a9641e1458b92c4cb099da6dfb9e38bb35e82d/data/helpers.d/utils#L239

Reply:
Which wasn't a reliable way to fix it either...
Anyway, if you try your last fix and if it indeed works, I love this one :)

Reply:
Yep I try to install/upgrade with and without this fix, it seems to work™